README.md

```diff
@@ -1,65 +1,45 @@
-88nine Radio Milwaukee
-=======================
+# 88nine Radio Milwaukee
 
 A node library for interacting with [88nine Radio Milwaukee](http://radiomilwaukee.org).
 
+```bash
+npm install --save 88nine
 ```
-npm install 88nine
-```
-
-## Playlists
-
-**Fetch** - Get a list of the 100 most recently played songs
 
-```javascript
-const playlist = require('88nine').playlist;
-playlist.fetch().then(function(songs) {
-  // do something with the songs
-  // Songs is an array of js objects with
-  // the following structure:
-
-  [{
-    playedAt: "2017-10-08T16:58:09.000Z",
-    artist: 'Abby Jeanne',
-    title: 'Cosmic Beings',
-    album: 'Cosmic Beings- Single',
-    composer: ''
-  }]
-});
+## How do I use it?
+
+```js
+const PlaylistStream = require('88nine/playlist-stream')
+
+// create a stream that checks for a new song once every 60 seconds (60,000ms)
+var stream = new PlaylistStream(60000)
+
+// this function will be called whenever the stream has data to be read. In this
+// case, that's whenever a new song is played.
+stream.on('data', (song) => {
+  // song is something like:
+  // {
+  //   album: 'Illinois',
+  //   artist: 'Sufjan Stevens',
+  //   duration: 520, // the duraiton in seconds
+  //   playedAt: new Date('2018-06-11T20:22:11.000Z'),
+  //   title: 'Casimir Pulaski Day',
+  //   isOriginalContent: false // is the song content made specifically for 88.9
```

Comment:
`isOriginalContent` is a bit of a tricky name, original content in the music industry usually refers to songs that are written by the artist or originally performed by the artist (i.e. not a cover). This could be confusing for any music-industry people using this data.

Reply:
Yeah, I'm not sure what to call it. I had it named `skipExternalLookup` before, but that doesn't make sense to anyone who isn't feeding the data into a service like spotify or musicbrainz.

Reply:
Should the logic for if a third-party lookup get moved out of this library? Maybe just have a list of 88nine program names in this lib? We don't have any mechanisms now for doing a lookup anyhow.

Reply:
yeah, we can do that - there's no reason why it needs to be in here

Reply:
Ok, it's removed now